configs/astro.json

```diff
@@ -0,0 +1,26 @@
+{
+  "index_name": "astro",
+  "start_urls": [
+    "https://docs.astro.build"
+  ],
+  "sitemap_urls": [
+    "https://docs.astro.build/sitemap.xml"
+  ],
+  "stop_urls": [],
+  "selectors": {
+    "lvl0": {
+      "selector": "",
+      "default_value": "Documentation"
+    },
+    "lvl1": ".content h1",
+    "lvl2": ".content h2",
+    "lvl3": ".content h3",
+    "lvl4": ".content h4",
+    "lvl5": ".content h5",
+    "text": ".content p, .content li"
+  },
+  "conversation_id": [
+    "1341434945"
```

Comment:
copied this from snowpack.json. Snowpack is my other project, so assuming this was okay to use for both.

`nb_hits` I wasn't sure about, and copied over as well from snowpack.dev

Reply:
No need to re-apply, I'll keep this `conversation_id`. For the `nb_hits` it's mostly indicative of the first crawl but we don't keep track of it.